lib/ex_unit/lib/ex_unit.ex

```diff
@@ -395,8 +395,9 @@ defmodule ExUnit do
   @doc since: "1.12.0"
   @spec async_run() :: Task.t()
   def async_run() do
+    options = persist_defaults(configuration())
+
```

Comment:
Had to be outside of `Task.async` to ensure `seed` was set when it's fetched in `test.ex`